config/exercise_data.json

```diff
@@ -1,6 +1,6 @@
 {
-    "two-fer": {
-        "code_file": "two_fer.ex",
-        "analyzer_module": "ElixirAnalyzer.ExerciseTest.TwoFer"
-    }
-}
\ No newline at end of file
+  "two-fer": {
+    "code_file": "two_fer.ex",
+    "analyzer_module": "ElixirAnalyzer.TestSuite.TwoFer"
+  }
+}
```

Comment:
Oh, I didn't notice this file before. Could you add the `pacman-rules` exercise here? Normally I would do it on my own, on my own branch, but since you're changing the module names, it makes more sense to add it on this branch.